apps/judicial-system/web/src/routes/PublicProsecutor/Indictments/Overview/Overview.strings.ts

```diff
@@ -38,4 +38,10 @@ export const strings = defineMessages({
       'Máli {caseNumber} hefur verið úthlutað til yfirlestrar á {reviewer}.',
     description: 'Notaður sem texti í tilkynningaglugga um yfirlesara.',
   },
+  changeReviewedDecisionButtonText: {
+    id: 'judicial.system.core:public_prosecutor.indictments.overview.change_reviewed_decision_button_text',
+    defaultMessage: 'Breyta ákvörðun?',
```

Comment:
I believe that this text is not supposed to say "Breyta ákvörðun?" but is rather a placeholder in the design because we had not decided how the users confirms their decision. I've asked Guðný about this in the Asana task, let's wait and see what she says.

Reply:
Updated the button name according to Guðný's comment [here](https://app.asana.com/0/inbox/1208883002434307/1208946541017356/1209068760596702)